passenger-service/src/main/java/com/me/passengerservice/controller/PassengerController.java

```diff
@@ -0,0 +1,55 @@
+package com.me.passengerservice.controller;
+
+import com.me.passengerservice.dto.request.PassengerRequest;
+import com.me.passengerservice.dto.response.PassengerListResponse;
+import com.me.passengerservice.dto.response.PassengerResponse;
+import com.me.passengerservice.service.PassengerService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/passengers")
+public class PassengerController {
+    private final PassengerService passengerService;
+
+    @GetMapping("/{id}")
+    public ResponseEntity<PassengerResponse> getPassengerById(@PathVariable Long id) {
+        return ResponseEntity.ok().body(passengerService.getPassengerById(id));
+    }
+
+    @GetMapping
+    public ResponseEntity<PassengerListResponse> getAllPassengers() {
+        return ResponseEntity.ok(passengerService.getAllPassengers());
+    }
+
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping
+    public ResponseEntity<PassengerResponse> createPassenger(@RequestBody PassengerRequest passengerRequest) {
+        return ResponseEntity.ok().body(passengerService.createPassenger(passengerRequest));
```

Comment:
Minor: no need in ResponseEntities, just return PassengerResponse

Also you have conflicting http status codes (CREATED mentioned in ResponseStatus annotation and OK in ResponseEntity)